### DISCUSSION TITLE
Future

### DISCUSSION THREAD

**Discussion by Adhiraj-C** [community/community#140116] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am thinking of uploading one project per day on github but I believe it would be tedious to do so since I am also doing one project per day for the CS50x course, is having a large repository on github worth it or no? I uploaded some basic codes today but I think some of the codes might not work properly, can anyone please give me any advice.

**Comment by feijoes** [comment]
Upload what you want, just don't create a large repository for the sake of it. Try creating some useful projects; they don't need to be complicated or big.

**Reply by Adhiraj-C** [reply]
I am uploading my progress is that fine
